test/Microsoft.Framework.Runtime.Tests/ProjectFacts.cs

```diff
@@ -373,7 +373,7 @@ public void CompilerOptionsAreSet()
         {
             var project = Project.GetProject(@"
 {
-    ""compilationOptions"": { ""allowUnsafe"": true, ""define"": [""X"", ""y""], ""platform"": ""x86"", ""warningsAsErrors"": true, ""optimize"": true }
+    ""compilationOptions"": { ""allowUnsafe"": true, ""define"": [""X"", ""y""], ""platform"": ""x86"", ""warningsAsErrors"": true, ""optimize"": true, ""strongNameKeyFile"" : ""c:\\keyfile.snk"", ""delaySign"" : true }
```

Comment:
Can we get a couple of variants for the `delaySign` values ('1`, '-1`, `True`, `""`)?